tests/Rector/DisallowedAttributesRector/DisallowedAttributesRectorTest.php

```diff
@@ -3,19 +3,19 @@
 use Symplify\SmartFileSystem\SmartFileInfo;
 
 it('can remove disallowed attributes', function () {
-    $this->doTestFileInfo(
+    $this->doTestFile(
         new SmartFileInfo(__DIR__ . '/Fixture/class_with_disallowed_attribute.php.inc')
     );
 });
```

Comment:
Should be able to replace these with just the string. i.e.

```suggestion
it('can remove disallowed attributes', function () {
    $this->doTestFileInfo(
    $this->doTestFile(
        __DIR__ . '/Fixture/class_with_disallowed_attribute.php.inc'
    );
});
```

Then we don't need the `symplify/smart-file-system` package dependency 👍🏻

Reply:
Thanks, will give it a try :)